CMakeLists.txt

```diff
@@ -93,6 +93,7 @@ if(NOT SwiftFoundation_MODULE_TRIPLE)
 endif()
 
 # System dependencies
+find_package(LibRT)
```

Comment:
For my context, is there a reason that's preventing us from fixing this on the dispatch side instead of applying a workaround to Foundation (and presumably any other project that imports dispatch via CMake in the toolchain)? Is there a way we can just add this to the exported config for dispatch?